static/app/views/settings/incidentRules/metricField.tsx

```diff
@@ -160,13 +159,6 @@ const MetricField = ({organization, columnWidth, inFieldLabels, ...props}: Props
 
       return (
         <React.Fragment>
-          {!inFieldLabels && (
-            <AggregateHeader>
```

Comment:
we no longer need this as all renditions of the metric alert builder use `inFieldLabels` or no labels for the metric field